_episodes/09-pull-requests.md

```diff
@@ -192,99 +291,224 @@ Largest City: Paris
 {: .output}
 
 Next we will add and commit these changes.
-Then we will push them to our forked version of the repo.
+Then we can push them to our forked version of the repo.
 
 ~~~
 $ git add france.txt
 $ git commit -m "Added largest city to france file"
-$ git push origin master
+$ git push origin addFrance
 ~~~
 {: .bash}
 
+~~~
+Counting objects: 3, done.
+Delta compression using up to 4 threads.
+Compressing objects: 100% (3/3), done.
+Writing objects: 100% (3/3), 387 bytes | 0 bytes/s, done.
+Total 3 (delta 0), reused 0 (delta 0)
+To https://github.com/sstevens2/countries.git
+   31aa2e3..609acfe  addFrance -> addFrance
+~~~
+{: .output}
+
 Now if we reload the pull request.  The new commit was added to that pull request.
```

Comment:
(Legacy lines) My suggestion is: "If we reload the pull request, we'll see that the new commit was added to the pull request and the changes have been automatically updated."